src/css/hero.css

```diff
@@ -0,0 +1,245 @@
+    margin-top: 32px;
+}
+
+.hero-buttons {
+    display: flex;
+    gap: 14px;
+}
+
+.hero-button-buy {
+    background-color: var(--orange);
+    color: var(--white);
+    font-family: var(--font);
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: normal;
+    cursor: pointer;
+    display: inline-flex;
+    padding: 12px 28px;
+    align-items: flex-start;
+    gap: 14px;
+    min-width: 120px;
+    border-radius: 100px;
+    border: none;
+    margin: 0;
+    white-space: nowrap;
+    transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+.hero-button-buy:hover,
+.hero-button-buy:focus {
+    background-color: var(--background-color);
+    color: var(--orange);
+}
+
+.hero-button-how {
+    background: none;
+    color: var(--white);
+    font-family: var(--font);
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: normal;
+    cursor: pointer;
+    display: inline-flex;
+    padding: 12px 28px;
+    align-items: flex-start;
+    gap: 14px;
+    min-width: 159px;
+    border-radius: 100px;
+    border-width: 1px;
+    border-style: solid;
+    border-color: var(--white);
+    margin: 0;
+    white-space: nowrap;
+    transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1), border-color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+.hero-button-how:hover,
+.hero-button-how:focus {
+    color: var(--orange);
+    border-color: var(--orange);
+}
+
+.hero-button-scroll {
+    background: none;
+    color: var(--white);
+    font-family: var(--font);
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.28571;
+    cursor: pointer;
+    display: flex;
+    padding: 0;
+    margin: 32px auto 0 0;
+    align-items: center;
+    flex-shrink: 0;
+    gap: 14px;
+    border: none;
+    white-space: nowrap;
+    transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+.icon-arrow {
+    background-color: var(--white);
+    fill: var(--orange);
+    width: 28px;
+    height: 28px;
+    border-radius: 50%;
+    transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1), color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+.hero-button-scroll:hover,
+.hero-button-scroll:focus,
+.icon-arrow:hover .hero-button-scroll,
+.icon-arrow:focus .hero-button-scroll {
+    color: var(--orange);
+}
+
+.hero-button-scroll:hover .icon-arrow,
+.hero-button-scroll:focus .icon-arrow,
+.icon-arrow:hover,
+.icon-arrow:focus {
+    background-color: var(--orange);
+    fill: var(--white);
+}
+
+@media screen and (min-width: 375px) and (max-width: 480px) {
+.hero-container {
+    max-width: 375px;
+    margin: auto;
+    padding: 0 20px 20px 20px;
+}
+
+.hero-text {
+    font-size: 20px;
+    margin: auto;
+    max-width: 100%;
+    font-size: 20px;
+}
+
+.hero-buttons {
+    display: flex;
+    gap: 14px;
+}
+
+.hero-button-scroll {
+    margin: 32px auto 0 0;
+}
+}
+
+@media screen and (min-width: 768px) {
+.hero-container {
+    max-width: 768px;
```

Comment:
teraz juz nie wiem, czy to jest poprawne, czy nie, skoro widze to juz ktorys raz

Reply:
A Ty jak miałaś w projekcie na zaliczenie? Co wpisywałaś w szerokości kontenerów?

Reply:
Starałam się nie określać sztywnie containera, ale jak już to jak miałam min-width to również dawałam min-width w containerze. Wydaje mi sie to najbardziej logiczne, bo w przeciwnym razie jak ma to dzialac jak jest media min widthL 768, a w containerze maxL 768

Reply:
Mogę zobaczyć z min-width, ale jak wtedy ustawiałam nie na sztywno to miałam poprawić i potem przeszło. A zobaczę jak to będzie wyglądać

Reply:
Ogólnie w tych szerokościach maksymalnych  kontenerów to chyba chodzi o to, że one pozostają bez zmian, a rozszerza się miejsce po bokach. Chociaż jeśli chodzi o hero to może rzeczywiście się nie opłaca,  szczególnie jak rzeczy nie są całkowicie wyśrodkowane tylko coś jest po bokach. Chyba najbardziej istotne to jest przy tych wielkich ekranach, żeby sobie człowiek karku nie skręcił przy czytaniu ;d Pokasowałam wszystkie ograniczenia, najwyżej jak trzeba będzie to się potem dołoży. W ogóle zawiesiło mi się coś, nie pokazało błędu ani zerwania połączenia, a stanęło w takim miejscu jak próbowałam tą listę bez flexa. Wcześniej zrobiłam commita, żeby w razie co wrócić jak zepsuję i jak już zwątpiłam i cofnęłam zmiany to mi się wszystko rozjechało i nieważne co robiłam, to tak jakby pokazywało tylko wybrane zmiany, ale nie wszystkie, dlatego nie od razu ogarnęłam, że coś nie tak. Już miałam pobrać kopię z Githuba i nanosić zmiany od nowa, ale sprawdzałam różnice z tamtym plikiem i nie było znaczących, które by miały wpłynąć tak na wygląd. Wkurzyłam się i zrobiłam commita, bo stwierdziłam, że i tak nic nie tracę i zresetowałam kompa i o dziwo jak znowu włączyłam już było ok. Także jakby Ci się coś takiego zrobiło, to wiesz co robić ;)